internal/connect/command_custom_plugin_list.go

```diff
@@ -23,12 +28,12 @@ func (c *customPluginCommand) newListCommand() *cobra.Command {
 		Example: examples.BuildExampleString(
 			examples.Example{
 				Text: "List custom connector plugins in the org",
-				Code: "confluent connect custom-plugin list --cloud aws",
+				Code: "confluent connect custom-plugin list --cloud AWS",
```

Comment:
We still recommend passing lowercase values. We will uppercase it before passing it to the backend.
```suggestion
				Code: "confluent connect custom-plugin list --cloud aws",
```